### PR TITLE
Added new backup information

### LINKED ISSUE #517: Guides: Update create backup guide
The backup system was updated with https://github.com/tachiyomiorg/tachiyomi/pull/3936

To update
- Create backup (new backup and legacy)
- Restore

To add
- Difference between legacy backup and new backup

### PR DESCRIPTION
**Pull request by loocool2**
Closes #517 and #531 

Don't know if we should mention TachiEH still since it's so dead

### CODE REVIEW

**Comment by arkon** [comment]
I think we removed EH elsewhere, right?

I also don't think AZ has the new backups yet but I could be wrong.

**Comment by jobobby04** [comment]
AZ has them in the dev builds I believe

File: src/help/guides/creating-backups.md [hunk 1]
```diff
@@ -14,14 +14,35 @@ Backups can hold the following information:
...
+
+
 ## Restoring
 
-Restoring is done from the <Navigation item="settings_backup"/> settings. To restore without issues be sure to:
+Restoring is done from the <Navigation item="settings_backup"/> settings. 
+
+To restore a backup without issues be sure to: 
+
+* Log into the <Navigation item="settings_tracking"/> services that you use.
```
Comment (ghostbear) [inline src/help/guides/creating-backups.md:43]: Shouldn't this be past tense, we are talking about the tracking services they used with the backup. Not what they currently use, which could also be different from what they previously used

So, maybe something along these lines
```suggestion
* Log into the <Navigation item="settings_tracking"/> services that you used.
```
or
```suggestion
* Log into the <Navigation item="settings_tracking"/> services that you previously used.
```
or
```suggestion
* Log into the <Navigation item="settings_tracking"/> services that you used with the backup.
```

This is also present in the legacy backup list for restoring

File: src/help/guides/creating-backups.md [hunk 2]
```diff
@@ -14,14 +14,35 @@ Backups can hold the following information:
...
+1. **Categories**
+1. **Read chapters**
+1. **Tracking settings**
+1. **Reading history**
+
+::: guide
+You can do this by going to <Navigation item="more"/> → <Navigation item="settings"/> → <Navigation item="settings_backup"/> and then selecting **Create legacy backup** and select a location to save it. 
+:::
+
+Legacy Backups are intended for versions of Tachiyomi older than 0.10.7 or versions of Tachiyomi based on versions older than 0.10.7.
```
Comment (ghostbear) [inline src/help/guides/creating-backups.md:34]: Couldn't this be written like this, or does it lose clarity?
```suggestion
Legacy Backups are intended for versions of Tachiyomi, or forks based on versions, older than 0.10.7
```

File: src/help/guides/creating-backups.md [hunk 3]
```diff
@@ -14,18 +14,39 @@ Backups can hold the following information:
 1. **Read chapters**
 1. **Tracking settings**
 1. **Reading history**
+1. **Manga information** such as Author, Artist, Date Added to Library , Selected Viewer, etc.
```
Comment (arkon) [inline src/help/guides/creating-backups.md:17]: Nitpick: remove extra space between `Library` and `,`

**Review by arkon — APPROVED** [review]

**Pull request merged**